.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET="Q948573-4958EPOIRJG;LSKFGJQ948TU49387OIJ;LJT9J4"
```

Comment:
For future reference, You want to avoid pushing secret keys to github by storing .env in .gitignore.